setup.py

```diff
@@ -4,45 +4,60 @@
 import versioneer
 
 
-DISTNAME = 'qrisk'
-DESCRIPTION = "qrisk is a Python library with performance and risk statistics commonly used in quantitative finance"
-LONG_DESCRIPTION = """qrisk is a Python library with performance and risk statistics
-commonly used in quantitative finance by `Quantopian Inc`_. It is used by `Zipline`_ open source backtesting library as well as `pyfolio`_.
+DISTNAME = "qrisk"
+DESCRIPTION = """qrisk is a Python library with performance and risk statistics
+commonly used in quantitative finance"""
+LONG_DESCRIPTION = """qrisk is a Python library with performance and risk
+statistics commonly used in quantitative finance by `Quantopian Inc`_.
 
 .. _Quantopian Inc: https://www.quantopian.com
 .. _Zipline: http://zipline.io
 .. _pyfolio: http://quantopian.github.io/pyfolio/
 """
-MAINTAINER = 'Quantopian Inc'
-MAINTAINER_EMAIL = 'opensource@quantopian.com'
-AUTHOR = 'Quantopian Inc'
-AUTHOR_EMAIL = 'opensource@quantopian.com'
-URL = "https://github.com/quantopian/pyfolio"
+MAINTAINER = "Quantopian Inc"
+MAINTAINER_EMAIL = "opensource@quantopian.com"
+AUTHOR = "Quantopian Inc"
+AUTHOR_EMAIL = "opensource@quantopian.com"
+URL = "https://github.com/quantopian/qrisk"
 LICENSE = "Apache License, Version 2.0"
-VERSION = "0.1"
 
-classifiers = ['Development Status :: 4 - Beta',
-               'Programming Language :: Python',
-               'Programming Language :: Python :: 2',
-               'Programming Language :: Python :: 3',
-               'Programming Language :: Python :: 2.7',
-               'Programming Language :: Python :: 3.4',
-               'License :: OSI Approved :: Apache Software License',
-               'Intended Audience :: Science/Research',
-               'Topic :: Scientific/Engineering',
-               'Topic :: Scientific/Engineering :: Mathematics',
-               'Operating System :: OS Independent']
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "License :: OSI Approved :: Apache Software License",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Mathematics",
+    "Operating System :: OS Independen"
+]
+
+
+test_reqs = [
+    "nose>=1.3.7",
+    "nose_parameterized>=0.5.0"
+]
+
 
-install_reqs = [
+requirements = [
     'numpy>=1.9.1',
     'pandas>=0.15.0',
```

Comment:
actually, we might be fine with the way we are using resample. I would be very suprised if this actually worked with 0.15 though, it should be at least >= 0.16.1